File: packages/rust/tests/fixtures/one_function/typescript/clean/methods.ts
export class Widget {
  private size: number;

  constructor(size: number) {
    this.size = size;
  }

  grow(amount: number): number {
    this.size += amount;
    return this.size;
  }

  shrink(amount: number): number {
    this.size -= amount;
    return this.size;
  }
}

File: packages/rust/tests/fixtures/one_function/rust/clean/methods.rs
pub struct Widget {
    size: u8,
}

impl Widget {
    pub fn grow(&mut self, amount: u8) -> u8 {
        self.size += amount;
        self.size
    }

    pub fn shrink(&mut self, amount: u8) -> u8 {
        self.size -= amount;
        self.size
    }
}
